src/components/GameSpecBuilder.js

```diff
@@ -29,6 +29,8 @@ const GameSpecBuilder = (props) => {
           setBoardSize={setBoardSize}
           setItems={setItems}
           getItems={getItems}
+          getPieces={getPieces}
```

Comment:
Instead of using both items and pieces, merge them to one, so that it is easier to maintain state.

Reply:
Sure, thanks